miscelanea/004-route-traffic/README.md

```diff
@@ -0,0 +1,520 @@
+# Redirect traffic to the cluster
+
+The goal of this proof of concept is to investigate the traffic to our pods
+in different scenarios:
+
+- [x] **[Scenario 1 - Inside the pod](#scenario-1---inside-the-pod)**: A client container should be able to connect the
+  different ports of the pod.
+- [x] **[Scenario 2 - Intra Pod](#scenario-2---intra-pod)**: A client in a secondary pod in the same node
+  should be able to connect to the different pod ports.
+- [x] **[Scenario 3 - Intra Pod different Nodes](#scenario-3---intra-pod-different-nodes)**: A client in a secondary pod in a
+  different node should be able to connect to the different pod ports.
+- [ ] **[Scenario 4 - External access](#scenario-4---external-access)**: Allowing access from out of the cluster to the
+  cluster.
+  - [x] **[nodePort](#nodePort)**.
+        - [x] It worked on CodeReadyContainers with no changes.
+        - [x] It worked on OpenStack cluster by adding two rules to the security group.
+  - [ ] **[externalIP + NodePort](#externalips--nodeport)**.
+        - [x] It worked on CodeReadyContainers by assigning an extra IP to the VM.
+        - [ ] Pending to run the same in an OpenStack cluster.
+  - [ ] **[externalIP + LoadBalancer](#externalip--loadbalancer)**.
+        - [ ] It should work with OpenStack Victoria.
+- [ ] **[Scenario 5 - Intracluster](#scenario-5---intracluster)**: A natural extension from the above is using the
+  services from another cluster so that we close the circle for the hybrid cloud scenario in terms of network traffic.
+      - [ ] Deploy in cluster 1 and use the services from cluster 2.
+      - [ ] Deploy in cluster 1 and cluster 2; cluster 2 is federated to cluster 1.
+
+The main pod is a set of containers running an echo server using different
+ports and transport layer (except for 80 and 443 ports which is running a
```

Comment:
Check that his match with the proof of concept. Probably the http-echo service is not sued eventually.

Reply:
It is not used eventually.